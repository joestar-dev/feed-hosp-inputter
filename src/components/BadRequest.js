import React, { Component } from 'react';
import { Link, NavLink } from 'react-router-dom';

const BadRequest = () => (
  <div>
    <div>Bad Request <Link to="/" activeClassName="is-active">Go Home</Link></div>
  </div>
);

export default BadRequest;