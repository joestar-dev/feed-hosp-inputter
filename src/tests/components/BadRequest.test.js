import React from 'react';
import { shallow } from 'enzyme';
import BadRequest  from '../../components/BadRequest';

test('should render BadRequest corectly', () => {
  const wrapper = shallow(<BadRequest />);
  expect(wrapper).toMatchSnapshot();
});