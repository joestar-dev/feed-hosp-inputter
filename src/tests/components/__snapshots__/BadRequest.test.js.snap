// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render BadRequest corectly 1`] = `
<div>
  <div>
    Bad Request 
    <Link
      activeClassName="is-active"
      replace={false}
      to="/"
    >
      Go Home
    </Link>
  </div>
</div>
`;
